vendors/espressif/manifest.cmake

```diff
@@ -1,5 +1,7 @@
 set(
     AFR_MANIFEST_SUPPORTED_BOARDS
+    esp32_plus_ecc608a_devkitc
+    esp32_plus_ecc608a_wrover_kit
```

Comment:
Up until now we do not have `wrover_kit` based on `ecc608a`, so you might want to remove this entry

Reply:
Okay, I will remove these entries.

Reply:
@mahavirj Removed these entries.